docs/docs/cli-commands.md

```diff
@@ -74,7 +74,6 @@ We use Babel to transpile the api side into `./api/dist` and Webpack to package
 | Arguments & Options | Description                                                                                                                                                                 |
 | :------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `side`              | Which side(s) to build. Choices are `api` and `web`. Defaults to `api` and `web`                                                                                            |
-| `--stats`           | Use [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) to visualize the size of Webpack output files via an interactive zoomable treemap |
```

Comment:
Maybe mention a vite-compatible alternative to this somewhere in the docs (unless you already do further down, and I just haven't gotten that far yet)

Reply:
It wouldn't fit in to this document here, in the CLI reference. 

Now that we're very close to standard Vite - any regular problem you can just google "how to analyse bundle with Vite", etc.

Reply:
You're totally right. It shouldn't be in the cli docs.
And lets not add anything to the configuring vite section either right now. It's more important to spend time on getting v6 out the door!